main/src/ui/occupant_menu/list_row.vala

```diff
@@ -19,6 +22,8 @@ public class ListRow : Object {
         main_grid = (Grid) builder.get_object("main_grid");
         picture = (AvatarPicture) builder.get_object("picture");
         name_label = (Label) builder.get_object("name_label");
+        status = (Label) builder.get_object("status_label");
+        status.label = "(unknown)";
```

Comment:
Hmm it doesn't seem like it should be like that
![image](https://github.com/mxlgv/dino/assets/51446645/90c02ed1-b543-4884-a578-b08c9ee8aae8)

Reply:
Will be fixed when adding online/offline icons